src/Components/test/E2ETest/Tests/EventTest.cs

```diff
@@ -133,6 +134,7 @@ public void PreventDefault_DotNotApplyByDefault()
         }
 
         [Fact]
+        [Flaky("https://github.com/aspnet/AspNetCore-Internal/issues/1987")]
```

Comment:
The flakiness is due to a product issue. Marking it as such until we fix the underlying bug.